string file
string name
geometry_msgs/Pose pose
string frame_id
---
bool success
